interpreter/binary/encode.ml

```diff
@@ -426,6 +426,7 @@ let encode m =
       | TableImport t -> u8 0x01; table_type t
       | MemoryImport t -> u8 0x02; memory_type t
       | GlobalImport t -> u8 0x03; global_type t
+      | EventImport x -> u8 0x04; vu32 0x00l; var x
```

Comment:
Similar to the decoder, I'd introduce an `event_type` encoding function.

Reply:
Done